calculator/src/components/GrowthInput.js

```diff
@@ -1,57 +1,85 @@
 import React,{useState} from "react";
-import "./styles.scss"
+import {Bar} from 'react-chartjs-2';
+import {Form} from 'react-bootstrap'
 
 export default function GrowthInput(props) {
   const [value, setValue] = useState("");
   const [rate, setRate] = useState("");
+  const [data, setData] = useState("");
 
   function submitData(){
-    const data = []
-    for(let i = 1;i <= 10;i++){
-      if (i === 1){
-        data.push(Number(value))
-      }
-      else{
-        data.push(data[i-2] + data[i-2]*(Number(rate)/100))
+    const growth = []
+    //later labels can be switched with days or months or years
+    const labels = []
+    if(value.length && rate.length){
+      for(let i = 1;i <= 10;i++){
```

Comment:
I'm curious to know why 10 is hardcoded here when there is the use of .length just above? Perhaps this is something that is lost due to the unavailable deployment of the project for context?

Reply:
I just calculate it for next 10 cycles but as the comment says above with more time I can choose to calculate growth by month week or day and the value won't be 10 instead the data array size I will be using.